doc/tutorials/gpu-passthru.rst

```diff
@@ -66,11 +66,11 @@ Passthrough the GPU to Guest
 
      cp /usr/share/acrn/samples/nuc/launch_win.sh ~/install_win.sh
 
-#. Modify the ``install_win.sh`` script to specify the Windows image you use.
+2. Modify the ``install_win.sh`` script to specify the Windows image you use.
 
-#. Modify the ``install_win.sh`` script to enable GVT-d.
+3. Modify the ``install_win.sh`` script to enable GVT-d.
```

Comment:
I see what you intended (what previously were separate steps should be part of step 3.  That's done by indenting these lines so they're part of step 3, so here's the right way to write this:

```
3. Modify the ``install_win.sh`` script to enable GVT-d:

   Add the following commands before ``acrn-dm -A -m $mem_size -s 0:0,hostbridge \``

   ::

     gpudevice=`cat /sys/bus/pci/devices/0000:00:02.0/device`

     echo "8086 $gpudevice" > /sys/bus/pci/drivers/pci-stub/new_id
     echo "0000:00:02.0" > /sys/bus/pci/devices/0000:00:02.0/driver/unbind
     echo "0000:00:02.0" > /sys/bus/pci/drivers/pci-stub/bind

   Replace ``-s 2,pci-gvt -G "$2" \`` with ``-s 2,passthru,0/2/0,gpu \``

4. Run ``launch_win.sh``.
```